examples/electrostatic-abb/electrostatic-abb.cpp

```diff
@@ -137,8 +152,6 @@ int main(int argc, char* argv[])
         solver->apply(b, x_clone);
```

Comment:
You can reduce the warmup here in line 150 to 1 instead of 3 to speed things up.